packages/api/spec/predictivemovement.yaml

```diff
@@ -35,8 +35,7 @@ paths:
           description: ID of the transport to which the itinerary is assigned
           required: true
           schema:
-            # type: string
-            $ref: "#/components/schemas/Transport/properties/transport_id"
```

Comment:
@iteadam 
It seems that during developing, one needs to generate Typescript types from the openapi specification and the npm package that we use does not seem to support this way to reference properties from a schema object.

Basically https://www.npmjs.com/package/openapi-typescript splits on `'/` and creates a path like 
```javascript
components["schemas"]["Transport"]["properties"]["transport_id"]
```
But the same package when it converts the components from openapi specification into interfaces does not include `properties` as a property inside `Transport` for example  so the path above fails.

I wasn't able to find a solution for that package but we could investigate more if we have time. 
Let's stick to the simple `type: string` notation and inform everyone in the team about it.